internal/api/server.go

```diff
@@ -228,9 +227,9 @@ func (s *server) Serve(ctx context.Context, l net.Listener) error {
 		}
 	}()
 
-	log.WithFields(logrus.Fields{
-		"tls": s.cfg.TLSConfig != nil,
-	}).Infof("Server is listening on %q", l.Addr().String())
+	log.WithValues(
+		"tls", s.cfg.TLSConfig != nil,
+	).Info("Server is listening", "port", l.Addr().String())
```

Comment:
This doesn't look quite right.

Reply:
Clarifying... `"port"` and `l.Addr().String()` don't look right as a key/val pair because `l.Addr().String()` may often evaluate to something like `0.0.0.0:8080`. Maybe we could use `"bindAddress"` as the key instead.

Reply:
Should be addressed in [54d2579](https://github.com/akuity/kargo/pull/1791/commits/54d257913008f1415189d1137eddd2cd44b1d237)